webhooks.md

```diff
@@ -2,19 +2,38 @@
 
 The Fidel API uses [webhooks](https://en.wikipedia.org/wiki/Webhook) to notify your application when relevant events happen in your account. You can utilise this functionality to receive data from events not triggered by direct API requests. Or to get the data in a service that is not responsible for making the API request but needs to consume the response.
 
-There are several webhooks available to use with the Fidel API, each corresponding to an event happening in your account: `brand.consent`, `card.linked `, `card.failed `, `program.status`, `location.status`, `transaction.auth`, `transaction.auth.qualified`, `transaction.clearing`, `transaction.clearing.qualified`, `transaction.refund` and `transaction.refund.qualified`. Each of them requires an HTTPS URL to be registered.
+There are several webhooks available to use with the Fidel API, each corresponding to an event happening either at **Program** or **Account** level. 
+
+The **Program** level events are: 
+- `card.linked `
+- `card.failed `
+- `program.status`
+- `location.status`
+- `transaction.auth`
+- `transaction.auth.qualified`
+- `transaction.clearing`
+- `transaction.clearing.qualified`
+- `transaction.refund`
+- `transaction.refund.qualified` 
+
+The **Account** level events are: 
+- `brand.consent`
+- `credits.balance.low`
+
+*Each event requires an HTTPS URL to be registered.*
 
 The Fidel API will notify your registered webhook URLs as the event happens, via a HTTP POST request. The HTTP request payload contains the Event object. For example, when a customer makes a payment with a linked Mastercard on a participating location, a `transaction.auth` event is sent in real-time to the specified webhook URL. The HTTP request payload contains the Transaction object.
 
 ---
 
 ## Creating Webhooks
 
-You can register up to ten webhook URLs per event type for each account. The Fidel API only accepts HTTPS URLs for webhook endpoints. Your webhook server must support HTTPS and have a valid certificate.
+You can register up to ten webhook URLs per event type for each Program/Account (depending on whether it is a **Program** or **Account** event). The Fidel API only accepts HTTPS URLs for webhook endpoints. Your webhook server must support HTTPS and have a valid certificate.
```

Comment:
Please change program and account to lower case.

Also, when talking about Fidel API, we don't need to add "The". So, the sentence should be: "Fidel API accepts only HTTPS..."